eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState.hs

```diff
@@ -530,11 +536,46 @@ pvCanFollow _ SNothing = True
 pvCanFollow (ProtVer m n) (SJust (ProtVer m' n')) =
   (m + 1, 0) == (m', n') || (m, n + 1) == (m', n')
 
+-- =============================
+
+-- | Incremental Stake, Stake along with possible missed coins from danging Ptrs
+data IncrementalStake crypto = IStake
+  { getStake :: !(Map (Credential 'Staking crypto) Coin),
+    dangling :: !(Map Ptr Coin)
```

Comment:
These are not really dangling anymore. Originally, we were only including the dangling pointers in this structure, but now it is _all_ the pointers. I think we need a new name. :)

Reply:
We now use credMap and ptrMap
`data IncrementalStake crypto = IStake
  { credMap :: !(Map (Credential 'Staking crypto) Coin),
    ptrMap :: !(Map Ptr Coin)
  }`